src/Tabs/Tabs.js

```diff
@@ -83,14 +102,16 @@ class Tabs extends React.Component {
 
 Tabs.propTypes = {
   activeTabId: PropTypes.string,
+  defaultTabId: PropTypes.string,
   children: PropTypes.oneOfType([
     PropTypes.arrayOf(Tab),
     PropTypes.objectOf(Tab)
   ]).isRequired
 };
 
 Tabs.defaultProps = {
-  activeTabId: undefined
+  activeTabId: undefined,
+  defaultTabId: undefined
```

Comment:
`activeTabId` = controlled mode
`defaultTabId` = set the initial tab for uncontrolled mode